addons/udes_stock/utils.py

```diff
@@ -0,0 +1,117 @@
+from urllib.parse import urljoin
+from odoo import _
+
+BASE_PRODUCT_IMAGE_URL = "/web/image/product.product/%i"
+
+
+def location_to_dict(location, **extras):
+    """Represent a location in a format that json can understand"""
+    # TODO: call from udes_simple_location_blocking and update with blocked location
+    # if location.u_blocked:
+    #     reason = location.u_blocked_reason or "no reason specified"
+    # else:
+    #     reason = False
+
+    loc = {
+        "barcode": location.barcode,
+        "name": location.name,
+        "id": location.id,
+        # "blocked": {"is_blocked": location.u_blocked, "reason": reason},
+        # TODO: enable when u_storage_format is implemented
+        # "storage_format": location.u_storage_format,
+    }
+    loc.update(extras)
+    return loc
+
+
+def product_to_dict(product, **extras):
```

Comment:
I think this method can be removed and used _get_info instead which is defined in udes_get_info module.

Reply:
This one can be completely removed as it uses the same field names as the product model. Will remove.

Reply:
As per discussion with Miquel, this will be removed and when workflows are ported a better way will be implemented to get this info. Most surely using __get_info() and not having a utils function just for this.